lib/travis/api/v3/renderer/subscription.rb

```diff
@@ -0,0 +1,5 @@
+module Travis::API::V3
+  class Renderer::Subscription < ModelRenderer
+    representation(:standard, :id, :valid_to, :first_name, :last_name, :company, :zip_code, :address, :address2, :city, :state, :country, :vat_id, :status, :source, :selected_plan)
```

Comment:
Hm, just a quick question. I cannot see the concurrency here. Is that intended? 🤔

Reply:
Yes, I didn't add it yet, we are not really using now.  Once we know if it's needed on the front-end we can add it